src/inmanta/module.py

```diff
@@ -887,7 +887,8 @@ def try_parse(x):
             comp_version = parse_version(comp_version.base_version)
             return cls.__best_for_compiler_version(modulename, versions, path, comp_version)
         else:
-            return versions[0]
+            LOGGER.warning("The Inmanta compiler is not installed")
+            return None
```

Comment:
perhaps is would be better to return versions[0] if len(versions) > 0 else None

Reply:
I agree. The change is applied.